README.md

```diff
@@ -8,16 +8,17 @@ This repository contains samples for usage of the Google Cloud Machine Learning
 
 ### CPU and GPU
 
-* [Keras Census](census/keras) - Keras single node model
-* [Census](census) - TensorFlow census models
-  * [Canned Estimator](census/estimator) - TensorFlow canned estimator model
-  * [Custom Estimator](census/customestimator) - TensorFlow custom estimator model
+* [Census](census) - Predict whether an individual earns at least $50,000 per year
```

Comment:
I would stay away from words "individual" and "50K". 50k is really specific implementation detail.